### ISSUE TITLE
detect modeset Xorg server crashes

### ISSUE THREAD

**Issue by brianjmurrell** [abrt/abrt#1348]
In a crash such as was filed at https://bugzilla.redhat.com/show_bug.cgi?id=1662548 and https://bugs.freedesktop.org/show_bug.cgi?id=109174 it would make for a better quality bug report if that kind of crash was detected and trapped and filed with abrt.

**Comment by ernestask** [comment]
But would it, really? I don’t see any of the bugs (or the one that was marked as being a duplicate of) having any response from the developer.

**Comment by ernestask** [comment]
Unless this is something that helps you debug issues, I don’t think it’s worth the effort.

**Comment by brianjmurrell** [comment]
Hard to say with this being so old now.  I might have had more of an opinion back a year ago when this was still fresh in my memory.  At this point, I suppose you can close it if you want.

**Comment by ernestask** [comment]
True, sorry about dropping the ball. My point here is one of providing convenience for the user versus providing value for the developer. It’s easier to persuade us when there is evidence that the data collected (and the kinds thereof) can directly contribute to the issue getting fixed.

**Comment by brianjmurrell** [comment]
I think the problem I was getting at was that an abrt report would know what kind of data to collect and do it.  The user doesn't know this and ends up filing a dud bug report that gets ignored.  Sadly too often when bug reports don't contain the needed data, rather than asking for it, the bug owner just moves on and ignores it.
